fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1673,4 +1673,7 @@ public class Config extends ConfigBase {
 
     @ConfField(mutable = true)
     public static boolean enable_new_publish_mechanism = false;
+
+    @ConfField
+    public static int tablet_checker_interval_seconds = 20;
```

Comment:
Since we want to make the conf naming style consistent for tablet schedule related logic, we better rename it to `tablet_sched_checker_interval_seconds` and put it with the other `tablet_sched_*` confs in the code, and because it's a new conf, we should add it in the documentation, you can see this pr : https://github.com/StarRocks/docs.zh-cn/pull/1455